docs/channel-usage.md

```diff
@@ -0,0 +1,21 @@
+# Channel usage
+
+## Introduction & Context
+
+The below channels are used by the Spaces library internally.
+
+### Space channel
+
+Each `Space` (as defined by the [`Space` class](/docs/class-definitions.md#space)) creates its own [Ably Channel](https://ably.com/docs/channels).
+
+The channel name is defined by the `name` of the Space and takes the form: `_ably_space_${name}`. The full name of a `channel` belonging to a `Space` called 'my_space' would therefore be `_ably_space_my_space`.
+
+### Cursors
+
+If any member of a `Space` subscribes to or sets cursor updates a channel is created for `cursors` updates.
+
+The channel name is defined by the channel name of the Space with the `_cursors` suffix, taking the form: `${space.getChannelName()}_cursors`. The full name of a channel belonging to a `Space` called 'my_space' would therefore be `_ably_space_my_space_cursors`.
+
+#### Events published
+
+1. `cursorUpdate` - a batch of cursor updates passed to [`set`](/docs/class-definitions.md#set).
```

Comment:
```suggestion
1. `update` - a batch of cursor updates passed to [`set`](/docs/class-definitions.md#set).
```

Reply:
It's not `cursorUpdate` but `update` instead. https://github.com/ably-labs/spaces/pull/120

Reply:
I actually think that we can probably get rid of the 'Events published' section from this file.

Reply:
hey @Srushtika , this is the name of the internal event emitted on the Ably Channel, not the event we emit in the API.